src/Bridge/Form/TimezoneTypeExtension.php

```
@@ -0,0 +1,30 @@
+    public function __construct(private readonly CurrentTimezoneProviderInterface $provider)
+    {
+    }
+
+    public static function getExtendedTypes(): iterable
+    {
+        return [DateTimeType::class, DateType::class, TimeType::class];
+    }
+
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        $resolver->setDefault('view_timezone', fn (Options $options): string => $this->provider->getTimezone()->value());
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$options'. (undefined)

(UnusedFormalParameter)